private/email/templates/accounts/inviteShopMember.html

```diff
@@ -75,7 +75,7 @@
                   <tr>
                     <td align="left" valign="top"><table width="100%" border="0" cellspacing="0" cellpadding="0">
                       <tr>
-                        <td width="223" height="40" align="center" valign="middle" bgcolor="#1999dd" style="font-family: 'Source Sans Pro', sans-serif; font-size:18px; font-weight:normal; line-height:18px;"><a href="{{url}}" style="text-decoration:none; color:#ffffff;">Get Started</a></td>
+                        <td width="223" height="40" align="center" valign="middle" bgcolor="#1999dd" style="font-family: 'Source Sans Pro', sans-serif; font-size:18px; font-weight:normal; line-height:18px;"><a href="{{url}}" style="text-decoration:none; color:#ffffff;">Visit {{shopName}}</a></td>
```

Comment:
"Visit {{shopName}}" isn't the most thrilling language. Should we leave it how it was, to say "Get Started"? Or change this to something else? Or keep it as "Visit {{shopName}}"?

Reply:
I'd agree about the language here. @rymorgan or @aaronjudd any thoughts?

"Get Started" could still work since there's a new link. 
There's no acceptance process right now I don't think, so there's nothing this user has to accept and this email is more confirmation that their permissions have been upgraded for the specified shop. Visit {shopName} kinda works in that sense too.